packages/slate-hotkeys/src/index.js

```diff
@@ -53,6 +51,20 @@ const isDeleteWordBackward = e =>
 const isDeleteWordForward = e =>
   IS_APPLE ? isDeleteWordForwardMac(e) : isDeleteWordForwardPC(e)
 
+const isDeleteCharBackward = e => {
+  if (isDeleteBackward(e) || (IS_APPLE && isDeleteCharBackwardMac(e)))
+    return true
+  if (e.key !== 'Backspace' && e.key !== 'Delete') return false
+  const other = [
+    isDeleteLineBackward,
+    isDeleteLineForward,
+    isDeleteWordBackward,
+    isDeleteWordForward,
+    isDeleteCharForward,
+  ].find(fn => fn(e))
```

Comment:
Hi, this method means:
if a keyevent includes backspace or delete, and is not captured by other handlers.  Then it will be captured by `deleteCharBackward`.  It ensures that all backspace/delete events are catched.   Therefore, it will prevents issues like https://github.com/ianstormtaylor/slate/issues/1966 and https://github.com/ianstormtaylor/slate/issues/1966

Reply:
Okay... but why would that be inside a function called `isDeleteCharBackward` then? It doesn't make sense. If anything, you'd make a new function called `isDelete` and use that.

But further, there's already one called `isContentEditable` that is [used in the Before plugin](https://github.com/ianstormtaylor/slate/blob/master/packages/slate-react/src/plugins/before.js#L396-L398) that results in `e.preventDefault()` being called. 

So maybe there's some other reason?

Reply:
The definition of `isContentEditable` does not include `ctrl+backspace`, `alt+backspace`, `alt+delete` and other `backspace | delete` events which are originally not captured by `isDelete???` method

```
const isContentEditable = e =>
  isBold(e) ||
  isDeleteCharBackward(e) ||
  isDeleteCharForward(e) ||
  isDeleteLineBackward(e) ||
  isDeleteLineForward(e) ||
  isDeleteWordBackward(e) ||
  isDeleteWordForward(e) ||
  isItalic(e) ||
  isRedo(e) ||
  isSplitBlock(e) ||
  isTransposeCharacter(e) ||
  isUndo(e)
```

I think it is a good idea to have isUndefinedDelete function, then I can, in after.js, capture the isUndefinedDelete with char backward. (Fixed in recent commit ba9aa5d)

Reply:
Oh, I see what you mean.  I will fix it with `isDelete` and `isContentEditable`

Reply:
Fixed~